configurations/es5-test.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    extends: [
+        '../configurations/es5.js',
```

Comment:
Ahh, ok, I thought it was accidental.
Yea, it's really easier to read that way and communicates the context. Agreed :)